### PR TITLE
Fixed(docs): add monika whatsapp notifier documentation

### LINKED ISSUE #178: Docs for Monika Notification Engine
- [x] Add how to use Monika Notification Engine in the docs
- [x] Add a CTA in the landing page to use Monika Notification Engine

### PR DESCRIPTION
**Pull request by ilmiawan**
# Monika PR
fix #178 
## What issue this PR is trying to solve?
Currently there is no documentation available for implementing Monika Whatsapp Notifier in Monika.
## How this PR solve the issue?
By adding Monika Whatsapp Notifier implementation document on Monika Notification page and
add CTA in landing page.
## How to test
- Build Monika
- run Monika
- access monika landing page and notification guide page

### CODE REVIEW

**Review by sapiderman — APPROVED** [review]
Doc **ONLY** changes are ok.  
Note: But included in this PR are the source changes from PR #207. I would separate the two PR as merging this will auto merge the #207 also. So merge this only after 207.

File: docs/src/pages/index.js [hunk 1]
```diff
@@ -129,9 +129,10 @@ const Home = ({ sponsors }) => {
                     <p className="mt-2  lg:mt-4 text-base xl:text-lg lg:leading-normal leading-6 text-gray-600">
                       Get notified of the incidents through your favourite tools
                       including by e-mail via SMTP, mailgun, and sendgrid,
-                      webhook, telegram, or through slack incoming webhook. More
-                      notification channels are coming including Microsoft
-                      Teams!
+                      webhook, telegram, slack incoming webhook or through your
+                      whatsapp number by registering to Monika Whatsapp
+                      Notifier. More notification channels are coming including
+                      Microsoft Teams!
```
Comment (nicnocquee) [inline docs/src/pages/index.js:135]: can delete this "More notification channels are coming including Microsoft Teams!"

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: docs/src/pages/guides/notifications.md [hunk 2]
```diff
@@ -298,4 +298,24 @@ Monika supports Discord. To enable notification via Discord, you must create a d
 | Type | Notification types                                            | `discord`                                                      |
 | Url  | The URL of the Discord Webhook that will receive notification | `https://discord.com/api/webhook/<webhook.id>/<webhook.token>` |
 
+## Monika Whatsapp Notifier
```
Comment (nicnocquee) [inline docs/src/pages/guides/notifications.md:301]: add (COMING SOON)

File: docs/src/pages/guides/notifications.md [hunk 3]
```diff
@@ -298,4 +298,24 @@ Monika supports Discord. To enable notification via Discord, you must create a d
...
+
+Monika now have its own notification channel, sent through your whatsapp number. To enable notification via Monika Whatsapp Notifier, you must create a Monika Notifier account first. Please consult [Intro to Monika Whatsapp Notifier](http://monika-whatsapp-staging.azurewebsites.net)
```
Comment (nicnocquee) [inline docs/src/pages/guides/notifications.md:303]: remove the link to notifier. we will use proper domain later.

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**